src/worker.cpp

```diff
@@ -143,37 +75,72 @@ void *Worker::jpeg_grabber(void *arg)
     LOG_DEBUG("Start jpeg_grabber thread.");
 
     StartHelper *sh = static_cast<StartHelper *>(arg);
-    int encChn = sh->encChn;
+    int jpgChn = sh->encChn - 2;
```

Comment:
Why -2? Can we calculate this from the stream number?

Reply:
encChn is the stream number and -2 is a calculation ?

Reply:
Yes, I meant as in is it always `stream2` What if its `stream3 in the future?

Reply:
We currently have encChn as an encoder channel.
This is also used as an index for the global_video array. 
A JPEG channel is also an encoder channel but due to the structure in a different array (global_jpeg) if we don't want to start with the global_jpeg array at 2 I have to subtract the video channels. 
But I am open to suggestions.